src/components/ec-modal/ec-modal.spec.js

```diff
@@ -33,6 +42,31 @@ describe('EcModal', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  it('should be focus trapped', async () => {
```

Comment:
This test is testing that after a one `tab` you will get focus on a close button. But in the name of test it is saying that it should trap focus. We should be testing in a test a behaviour that it declares, and here it is trapping focus, that means do not let another elements outside component get it while we are "tabing".

Reply:
so should just i change the assertion?

Reply:
I'd try to test the trapping functionality described in the name of the test. For that purpose, we could create an environment for our modal component, as you started on line 46. For example, div.#root with a `button` inside it, mount modal to the div.#root and then trigger the "tab" key, checking every time that `button` doesn't get a focus.